grim.go

```diff
@@ -58,12 +57,13 @@ func (i *Instance) PrepareRepos() error {
 	var topicARNs []string
 	for _, repo := range repos {
 		localConfig, err := getEffectiveConfig(configRoot, repo.owner, repo.name)
+		if err != nil {
+			return fatalGrimErrorf("Error with config for %s/%s. %v", repo.owner, repo.name, err)
```

Comment:
am seeing this 4 times in logs on auth_build when it encounters it.  curious why...